.limedeploy

```diff
@@ -0,0 +1,10 @@
+# Common params
+DEPLOY_LIB_NAME='WultraSslPinning'
+DEPLOY_MODE="gradle"
+
+# Gradle
+DEPLOY_GRADLE_PATH="."
+DEPLOY_GRADLE_PARAMS="clean build install bintrayUpload"
```

Comment:
@TomasKypta just validate this params passed to gradlew

Reply:
You can change build to `assembleRelease` should be faster